msos/WaitChains.cs

```diff
@@ -227,7 +230,7 @@ public List<UnifiedBlockingObject> GetManagedBlockingObjects(uint osThreadId)
         protected virtual UnifiedBlockingObject GetUnifiedBlockingObjectForHandle(UnifiedHandle handle)
         {
             return new UnifiedBlockingObject(handle.Value, handle.ObjectName, handle.Type);
-        }
+        }   
```

Comment:
Extraneous whitespace inserted. Can you please look at the diff with a diff tool to check for this kind of issues in the future?